### DISCUSSION TITLE
Headers for paths with `basePath` set to `false`

### DISCUSSION THREAD

**Discussion by jNaimXIII** [vercel/next.js#54780] [Help]
### Summary

In [the docs](https://nextjs.org/docs/app/api-reference/next-config-js/headers#headers-with-basepath-support) it is shown how you can set headers for a path and disable the `basePath`. If I do that, how am I supposed to access the `/without-basePath` route? It will just give me a `404` error. Is there a reason why we can set headers for routes that are un-accessible or is there a way to override routes to not follow the `basePath`?

**Comment by minhajul-im** [comment]
I'm facing the same problem bhai,
thanks for the post..
